packages/wonder-blocks-dropdown/src/components/checkbox.tsx

```diff
@@ -69,16 +67,16 @@ const styles = StyleSheet.create({
         borderRadius: 3,
         borderWidth: 1,
         borderStyle: "solid",
-        borderColor: offBlack50,
+        borderColor: semanticColor.border.strong,
```

Comment:
Out of curiosity, should the Checkbox in the dropdown always have the same style as the Checkbox component from the form package? Would it make sense to share styles between the 2? Or let them each evolve on their own? 

If they should always be the same styles, another alternative could be to use the Checkbox component from the form package directly!